src/qt/bitcoin.h

```diff
@@ -90,6 +90,8 @@ class BitcoinApplication: public QApplication
     /// Setup platform style
     void setupPlatformStyle();
 
+    interfaces::Node& node() { return m_node; }
```

Comment:
Add only when needed?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/19098#discussion_r456120469

> Add only when needed?

Thanks, removed here. It's added in https://github.com/bitcoin-core/gui/pull/35